bluepysnap/schemas/definitions/node.yaml

```diff
@@ -0,0 +1,76 @@
+title: Node file definitions
+description: schema definitions that apply to all node files
+$node_file_defs:
+  nodes:
+    maxProperties: 1
+    minProperties: 1
+    type: object
+    patternProperties:
+      "":
+        type: object
+        required:
+          - "0"
+          - node_type_id
+        additionalProperties: false
```

Comment:
I wonder if this is the case; we may have to figure out a way to be permissive in some cases.

Reply:
Yeah, according to my recent research jsonschema stuff, we should abstain from using `additionalProperties: false` unless it's a clear case. I need to review the cases I used it.